docs/backup-restore.md

```diff
@@ -1,134 +1,208 @@
+  <button class="tablinks" onclick="openTab(event, 'SQLite1', 'tabset-1')">SQLite</button>
+</div>
+
+<div id="Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL1" class="tabcontent" markdown="1">
+
+You can restore to any point within the backup retention period by using the automated backup feature.
+</div>
+<div id="MySQL1" class="tabcontent" markdown="1">
+
+Use the `mysqldump` command with the `--single-transaction` option.
+</div>
+<div id="PostgreSQL1" class="tabcontent" markdown="1">
 
 Use the `pg_dump` command.
+</div>
+<div id="SQLite1" class="tabcontent" markdown="1">
+
+Use the `.backup` command with the `.timeout` command as specified in [Special commands to sqlite3 (dot-commands)](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_)
+
+For an example, see [BASH: SQLite3 .backup command](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command).
+</div>
+</div>
+
+### Back up with explicit pausing
+
+Another way to create a transactionally consistent backup is to create a backup while a cluster of ScalarDB instances does not have any outstanding transactions. Creating the backup depends on the following:
+
+- If the underlying database has a point-in-time snapshot or backup feature, you can create a backup during the period when no outstanding transactions exist.
+- If the underlying database has a point-in-time restore or recovery (PITR) feature, you can set a restore point to a time (preferably the mid-time) in the pause duration period when no outstanding transactions exist.
+
+{% capture notice--info %}
+**Note**
+
+When using a PITR feature, you should minimize the clock drifts between clients and servers by using clock synchronization, such as NTP. Otherwise, the time you get as the paused duration might be too different from the time in which the pause was actually conducted, which could restore the backup to a point where ongoing transactions exist.
 
-#### SQLite (backup)
+In addition, you should pause for a sufficient amount of time (for example, five seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+{% endcapture %}
 
-Use the [`.backup` command](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_) along with `.timeout`.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-See also [this post](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command), for example.
+To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should implement the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB or use [ScalarDB Cluster (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/), which uses the Scalar Admin interface.
```

Comment:
```suggestion
To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should implement the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB or use [ScalarDB Cluster (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/), which implements the Scalar Admin interface.
```
The Admin interface is something we implement, not use.